src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -105,13 +113,22 @@ object SparkCassandraStress {
           |                             bulk: bulkSaveToCassandra
           |                             driver: saveToCassandra
           |                           **Note**: Folder for DSEFS writes will be named: keyspace.tablename (Default: ks.tab)
-          |                           dataset save methods:
+          |                           dataframe save methods:
           |                             driver: ds.write...save()
           |                             parquet: data format in DSEFS
           |                             text: data format in DSEFS
           |                             json: data format in DSEFS
           |                             csv: data format in DSEFS""".stripMargin}
 
+      opt[String]('u',"dataframeSaveMode") optional() action { (arg,config) =>
```

Comment:
And this could also be SaveMode, https://github.com/apache/spark/blob/master/sql/core/src/main/java/org/apache/spark/sql/SaveMode.java